<launch>
  <node pkg="nodelet" type="nodelet" name="standalone_nodelet"  args="manager" output="screen"/>

  <node pkg="nodelet" type="nodelet" name="SocketCANReaderNodelet" args="load osa_communication/SocketCANReaderNodelet standalone_nodelet" output="screen">
  </node>
</launch>